src/Client/ClientInterface.php

```diff
@@ -11,5 +11,5 @@
 
 interface ClientInterface extends BaseClientInterface
 {
-    public function sendAsyncRequest(RequestInterface $request): PromiseInterface|Promise;
```

Comment:
@htuscher Why did you have to do this modification?

Reply:
Because the method got renamed in Guzzle 7: https://github.com/guzzle/guzzle/blob/2779e868a00289e1b1fd854af030c43a06f4bcb4/src/ClientInterface.php#L39